examples/microservices/skaffold.yaml

```diff
@@ -9,8 +9,8 @@ build:
 deploy:
   kubectl:
     manifests:
-      - leeroy-web/kubernetes/*
-      - leeroy-app/kubernetes/*
+    - leeroy-app/kubernetes/*
```

Comment:
is this intentional? this will be copied over during release.

Reply:
oh shoot, looks like I accidentally committed a change from when I was messing around with things. I'll fix that